examples/basics/visuals/polygon_prototype.py

```diff
@@ -0,0 +1,49 @@
+from vispy import app
```

Comment:
all right.

This brings me back to the question I had - how do you use a Visual without a scenegraph? how does the Visual compute the transforms that it needs? do you manually call ```_prepare_transforms()```? or is there some other way to do it?